### PR TITLE
docs: add lighthouse-monitor, minor adjust list layout

### LINKED ISSUE #9701: Project list using lighthouse: lightmon
Hi there,
we've been working with one of our clients on continuous monitoring of their web performance and build a tool for that called *Lighthouse Monitor* (or lightmon for short). It features
* running lighthouse against a set of URLs with a set of presets (e.g. desktop, smartphone on fast connection, smartphone on slow connection),
* distributing the results to different places (e.g. prometheus feeding graphana or chatbots),
* saving the complete reports (including all traces),
* a webpage to compare any two saved reports with each other side-by-side, and
* automatic cleanup of saved data (e.g. to reduce the number of reports from hourly to daily or from daily to weekly after a specific timespan)

Lightmon can perform evaluations in parallel by sharding the data across different worker nodes (instead of the internal parallel option, which we found often skews the results) and therefore and has been running at Verivox for about a year now.

### PR DESCRIPTION
**Pull request by LarsKumbier**
**Summary**
* Adds one of my projects using lighthouse to the documentation
* some small fixes to the docs layout

**Related Issues/PRs**
fixes #9701

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -313,21 +313,22 @@ Other awesome open source projects that use Lighthouse.
...
+* **[lighthouse-jest-example](https://github.com/justinribeiro/lighthouse-jest-example)** - Gather performance metrics via Lighthouse and assert results with Jest; uses Puppeteer to start Chrome with network emulation settings defined by WebPageTest.
 * **[lighthouse-lambda](https://github.com/joytocode/lighthouse-lambda)** - Run Lighthouse on AWS Lambda with prebuilt stable desktop Headless Chrome.
-* **[lighthouse-magic-light](https://github.com/manekinekko/lighthouse-magic-light)** set the color of the MagicLight Bluetooth Smart Light Bulb based on Lighthouse score
+* **[lighthouse-magic-light](https://github.com/manekinekko/lighthouse-magic-light)** - Set the color of the MagicLight Bluetooth Smart Light Bulb based on Lighthouse score
 * **[lighthouse-mocha-example](https://github.com/rishichawda/lighthouse-mocha-example)** - Run Lighthouse performance tests with Mocha and chrome-launcher.
+* **[lighthouse-monitor](https://github.com/verivox/lighthouse-monitor)** - Lighthouse for companies. Run profiles against all your URLs. Send metrics anywhere you want, save all reports with automatic data retention, compare any two results in a WebUI.
```
Comment (brendankenny) [inline readme.md]:
```suggestion
* **[lighthouse-monitor](https://github.com/verivox/lighthouse-monitor)** - Run Lighthouse against all your URLs. Send metrics to any backend you want, save all reports with automatic data retention, and compare any two results in a web UI.
```

Comment (patrickhulce) [reply readme.md]: +1 to the removing `for companies` change, that made me think it was more of a enterprise service :)

very neat library thanks @LarsKumbier!

**Comment by LarsKumbier** [comment]
fine by me - thanks for the suggestions!

**Comment by paulirish** [comment]
Looks great. Really excited to get some  hands on experience with lighthouse-monitor

**Comment by LarsKumbier** [comment]
@paulirish Thanks for merging - if you try it out, I'd love some feedback and your thoughts on it.

**Pull request merged**